### ISSUE TITLE
Endpoint embedded in javascript code

### ISSUE THREAD

**Issue by frisso** [netgroup-polito/frog4-migration-orch#2]
In file
frog4-migration-orch/web_gui/static/js/functions.js

there's the following code at line 5:
  migration_orch_endpoint = "http://127.0.0.1:8083/v1

Shouldn't this be in a config file or so?

**Comment by piscoroma** [comment]
Yes, you are right. I wrote it during the tests and i forgot to move it.

Il 02/Lug/2017 15:29, "Fulvio Risso" <notifications@github.com> ha scritto:

**Comment by ivanocerrato** [comment]
Is this issue fixed now?

**Comment by piscoroma** [comment]
Fixed.

**Comment by frisso** [comment]
@piscoroma : in fact, data is still "embedded" in file "appConfig.js". Never mind, please just tell people that there's this file with this data by writing a proper section in the main README.md file.

**Comment by piscoroma** [comment]
I know, in fact i tried to put the configuration in a .json rather than in
a .js but it was not easy because in javascript the reading of a json file
is asyncronus. So i chose to put in a .js that was easier. Anyway ok, i'm
writing about it in the readme.

Il 12/Lug/2017 00:23, "Fulvio Risso" <notifications@github.com> ha scritto:

**Comment by frisso** [comment]
The problem, to my eyes, is not that you added the config in a .js. The biggest problem is that the file cannot be easily located. But, never mind, just document it and it's ok. Thanks.
